src/Things/Structures/MineFacility.h

```diff
@@ -13,7 +13,7 @@ class MineFacility: public Structure
 	typedef NAS2D::Signals::Signal<MineFacility*> ExtensionCompleteCallback;
 public:
 	MineFacility(Mine* mine);
-	
+
```

Comment:
I noticed this a little after pushing. This was the results of resolving a merge conflict. I had started on these changes days ago, but then put them aside. Meanwhile other changes modified these lines. The difference between the two branches, was whether or not a stray tab was left around.

Considering the tab should be removed anyway, I'm just going to leave it.